File: CodingBat/Python/Warmup_1/front_back.py
#given a string, return a new string where the first and last chars have been exchanged.
def front_back(str):
    x=""
    l=0
    for i in str:
        if l==0:
            l+=1
            x=x+str[len(str)-1]
        elif l == (len(str) -1):
            x=x+str[0]
            l+=1
        else:
            x=x+str[l]
            l+=1
    return x

File: CodingBat/Java/Warmup_1/frontBack.java
public class frontBack{
    // Given a string, return a new string where the first and last chars have been exchanged.
    
    public static String frontBack(String str){
        int end = str.length();
        if (end <2){
            return str;
        }
        int begin = 0;
        String x="";
        for (int i=1;i<end-1;i++){
            x+=str.charAt(i);
        }
        return str.charAt(end-1)+x+str.charAt(0);
    }
    public static void main(String[] args) {
        
    }
}